优化登录页面

ycyv opened the pull request:
1. 新增global.js文件
2. 新增global.css文件
3. 优化base.html和header.html，使得前端页面支持模板
4. 删除多余无效的global.css内的属性
5. 优化global.css内//与/**/注释符号混用问题，统一只是用/**/
6. 删除global.css文件末尾多余换行符
7. 优化global.js文件布局格式，删除所有/*-------------*/,使用空行进行函数分割
8. 取消base.html中的block titles的硬编码
9. 优化django{if}模板内的内容，不需要缩进
10. 修改账号密码输入框的提示信息

ycyv committed:
优化登录页面
1. 新增global.js文件
2. 新增global.css文件
3. 优化base.html和header.html, 使得前端页面支持模板

ycyv committed:
优化登录页面
1.新增global.js文件
2.新增global.js文件
3.优化base.html和header.html，使得前端页面支持模板

enfancemill commented:
下次提交的时候需要注意：
1. git log不要写重复项。
2. 避免使用全角字符， 比如git log第三行。
3. 序号之后的.和内容中间最好空一格。
4. 提交的时候不要把git log复制上来，最好你自己详细的描述下你做了什么改动。

enfancemill commented:
@ycyv 下次提交的时候需要注意：

git log不要写重复项。
避免使用全角字符， 比如git log第三行。
序号之后的.和内容中间最好空一格。
提交的时候不要把git log复制上来，最好你自己详细的描述下你做了什么改动。

enfancemill commented:
@ycyv 另外，你不是先做的注册页面么，为什么我看到提交上来的分支是feature-add_login_page？

enfancemill commented:
@ycyv  其实你可以多学习下 @245967906 ，他的代码写的真的超级棒！

enfancemill commented on template/login.html:
```diff
@@ -1,27 +1,21 @@
-        </tr>
-        <tr>
-            <td><input type="submit" value="登录"></td>
-            <td><a href="{{ api_register }}">没有帐号?立即注册</a></td>
```
@ycyv  这一行代码在新修改的页面里被删除了？是有什么原因么？如果是忘了的话，补上吧。

ycyv replied:
使用的是<input>标签 没有使用table标签，再下一个版本将会优化，这里先将就吧

ycyv replied:
哦哦  加上了  这句话

enfancemill commented on template/layouts/header.html:
```diff
@@ -0,0 +1,8 @@
+<div class = "header">
+    <img id="img-title" src="https://oerugfbxb.qnssl.com/wp-content/themes/Earthshaker-1/images/logo.png"/>
```
@ycyv 不要在我们的代码中引用别人的图片资源，把图片放到本地。如果对方设置了防盗链我们的图片可能会失效。

ycyv replied:
由于美工还没有设计出图标，这里就先不改动了吧？

enfancemill commented on template/login.html:
```diff
@@ -1,27 +1,21 @@
-    </table>
-</form>
+
+    <div  class= "login">
```
@ycyv 代码中不应该有多余的空格。

ycyv replied:
已修复

enfancemill commented on template/login.html:
```diff
@@ -1,27 +1,21 @@
+    <div  class= "login">
+        <form action="{{ api_login }}" method="POST">
+            {% if error_msg %}
+                <p id="error_msg">{{ error_msg }}</p>
```
@ycyv  if嵌套中的模板，是不需要进行缩进的。

ycyv replied:
已优化

enfancemill commented on template/login.html:
```diff
@@ -1,27 +1,21 @@
+            {% endif %}
+            <input name="next" type="hidden" value="{{ next_page }}">
+            <input placeholder = "磨坊登录账号"  type = "text" name = "username" id="username_id"><br>
+            <input placeholder = "磨坊登录密码" type = "password" name = "password" id ="password_id"></br>
```
@ycyv 这里不是写C语言，请不要再=号两边空格。很难看。

ycyv replied:
该问题再下一个版本集中修复

enfancemill commented on static/css/global.css:
```diff
@@ -0,0 +1,120 @@
+    padding: 10px 2px 20px 40px; //内边距，上（top）、右(right)、下(bottom)、左(left)
+    height: 44px;
+    width: ;
+    margin: auto;  /*外边距*/
```
注释风格请保持统一，单行注释不应该混用//与/**/。

ycyv replied:
已优化

ycyv replied:
css文件只支持/**/注释符号 不支持//      js文件两者都支持

enfancemill commented on static/css/global.css:
```diff
@@ -0,0 +1,120 @@
+{
+    padding: 10px 2px 20px 40px; //内边距，上（top）、右(right)、下(bottom)、左(left)
+    height: 44px;
+    width: ;
```
@ycyv 这个width什么鬼？

ycyv replied:
多余无效的width属性已删除

enfancemill commented on static/css/global.css:
```diff
@@ -0,0 +1,120 @@
+
+
+
+
```
@ycyv 文件末尾不需要这样多余的空格，还有请统一括号的风格。将起始大括号与选择器放置在同一行。

ycyv replied:
已删除多余空格，‘还有请统一括号的风格。将起始大括号与选择器放置在同一行’  我们再商议一下风格

enfancemill commented on template/layouts/base.html:
```diff
@@ -1,20 +1,27 @@
-    {% block header %}
-    {% include 'layouts/header.html' %}
+    {% block titels %}
+    <title>童年磨坊登录页面</title>
```
@ycyv 请重新规划这里的布局，base模板中硬编码title将会在其他地方无法进行复用。

ycyv replied:
删除硬编码title,以后针对不同页面，需要重写 block titels

enfancemill commented on static/js/global.js:
```diff
@@ -0,0 +1,65 @@
+         $("#background1").fadeIn("slow");
+         $("#background2").fadeOut("slow");
+    });
+/*--------------------------------------------------------------------------------------------------*/
```
@ycyv 代码的分隔，依赖的是空行而不是这种分割线。

ycyv replied:
已优化，删除所有/*-------------*/  使用空行进行函数分割

enfancemill commented on template/login.html:
```diff
@@ -1,27 +1,21 @@
+                <p id="error_msg"></p>
+            {% endif %}
+            <input name="next" type="hidden" value="{{ next_page }}">
+            <input placeholder = "磨坊登录账号"  type = "text" name = "username" id="username_id"><br>
```
@ycyv  把魔坊这两个字相关的内容删除掉吧。

ycyv replied:
已优化

enfancemill replied:
@ycyv 代码中还有很多处使用到了魔坊的内容，都改下吧。

enfancemill reviewed (commented):
这里列出的内容，先修改下吧。

ycyv committed:
优化前端整体代码风格
1. 删除多余无效的global.css内的属性
2. 优化global.css内//与/**/注释符号混用问题，统一只是用/**/
3. 删除global.css文件末尾多余换行符
4. 优化global.js文件布局格式，删除所有/*-------------*/,使用空行进行函数分割
5. 取消base.html中的block titles的硬编码
6. 优化django{if}模板内的内容，不需要缩进
7. 修改账号密码输入框的提示信息

ycyv committed:
解决pull过程冲突

ycyv commented:
请尽快查阅

enfancemill commented on template/login.html:
```diff
@@ -1,27 +1,21 @@
-        </tr>
-        <tr>
-            <td><input type="submit" value="登录"></td>
-            <td><a href="{{ api_register }}">没有帐号?立即注册</a></td>
```
@ycyv 这里指的是23行的注册页跳转功能。

enfancemill commented on template/layouts/header.html:
```diff
@@ -0,0 +1,8 @@
+<div class = "header">
+    <img id="img-title" src="https://oerugfbxb.qnssl.com/wp-content/themes/Earthshaker-1/images/logo.png"/>
```
建议是还是先改过来，图片我会催下我们的设计同学。

enfancemill replied:
@ycyv 建议是还是先改过来，图片我会催下我们的设计同学。

enfancemill replied:
0.0

enfancemill commented on template/login.html:
```diff
@@ -1,27 +1,21 @@
-    </table>
-</form>
+
+    <div  class= "login">
```
@ycyv 这里，样式为什么直接嵌入在html中而不是单独的放在css里？

enfancemill commented on template/login.html:
```diff
@@ -1,27 +1,23 @@
-
+{% load static %}
+{% block titles %}
+<title>宝宝云运维管理平台登录页面</title>
```
这里建议直接使用登录就可以了，没必要那么复杂。

ycyv committed:
login.html页面增优化
1 增加'没有账号？立即注册字段

ycyv commented:
请尽快查阅
